docs/source/example_notebooks/dowhy_simple_example.ipynb

```diff
@@ -17,6 +17,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import os, sys\n",
```

Comment:
this is only for testing locally. remove this from the committed file. Just simple "import dowhy"